roles/tendrl-server/tasks/firewalld.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
The other detail here to check is why we don't use firewalld services we included in some tendrl packages.